lib/Cake/View/Helper/FormHelper.php

```diff
@@ -1880,7 +1883,7 @@ public function postLink($title, $url = null, $options = array(), $confirmMessag
 		if (isset($options['data']) && is_array($options['data'])) {
 			foreach (Hash::flatten($options['data']) as $key => $value) {
 				$fields[$key] = $value;
-				$out .= $this->hidden($key, array('value' => $value, 'id' => false));
+				$out .= $this->hidden($key, array('value' => $value, 'id' => false, 'secure' => static::SECURE_SKIP));
```

Comment:
The CSRF token of the postlink form is created without considering FormHelper's `fields` property, a few lines after the cited change: 

    $out .= $this->secure($fields);

The change just removes them from the outer form hash, not from its own.